README.md

```diff
@@ -39,8 +39,8 @@ Some sample files are located in `./input_json_files` and `./output_csv_files`
 
 ## New Instruction
 
-- L'objectif global du test : Lorsque l'on execute la cmd "\$ruby le_programme_d'entrée.rb", tous les fichiers _.json qui sont dans /input_json_files, soient convertis (ou non) en _.csv dans /output_csv_files.
-  _Les fichiers Input peuvent ne pas être sous format JSON ET/OU formatés différement._
+- L'objectif global du test : Lorsque l'on execute la cmd "\$ruby le*programme_d'entrée.rb", tous les fichiers *.json qui sont dans /input*json_files, soient convertis (ou non) en *.csv dans /output*csv_files.
+  \_Les fichiers Input peuvent ne pas être sous format JSON ET/OU formatés différement.*
```

Comment:
Tu peux le tester au travers de tests Rspec si tu veux :) mais ce n'est pas un critère déterminant pour le test, il s'agit d'un exemple d'erreur qui puisse être gérée dans des tests unitaires

Reply:
J'ai commencé à chercher comment récupérer les les fichiers en utilisant 
Dir.children("#{FILE_PATH}")

def find_file
  file_list = Dir.children("#{FILE_PATH}")
end

Cela me permet de store un array avec les noms des fichiers ["users.json", "new_story.json"]

def select_json
  Pour chaque element de file_list
    sélectionner tous les elements se terminant par .json et l'enregistrer dans une variable
  Fin de la selection
end

Dig into Dir and File doc to select all the files from dir and convert it to csv.
case *json 
  => ouvrir chaque element avec JSON.parse(File.read(...))
case !*json 
   => File.read("....")

Je suis toujours en cours de reflexion pour cette partie du test.